### ISSUE TITLE
powerdevil: hash mismatch

### ISSUE THREAD

**Issue by sjau** [NixOS/nixpkgs#96949]
**Describe the bug**
Rebuidling the system on nixos unstable small fails because of powerdevil patch hash mismatch:

**To Reproduce**
1. Use Powerdevil
2. Set to nixos-unstable small
3. Rebuild

```
building '/nix/store/zh01731y5g39plzwfklr9p0cls64jgvi-unit-script-docker-prune-start.drv'...

trying https://invent.kde.org/plasma/powerdevil/-/commit/c7590f9065ec9547b7fabad77a548bbc0c693113.patch
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  8903    0  8903    0     0  45891      0 --:--:-- --:--:-- --:--:-- 45891
building '/nix/store/anslprwnf3rhp6vwbaz1zl5zwrzjj7x7-set-environment.drv'...
hash mismatch in fixed-output derivation '/nix/store/cw5wy5s1xbjn5gsx89dgqz12xq0hgchp-c7590f9065ec9547b7fabad77a548bbc0c693113.patch':
  wanted: sha256:0vj70mhx6qhvbh4vn9qk9ir5w4s2m76hw2lsxmh3ibgsydz4yilz
  got:    sha256:077whhi0jrb3bajx357k7n66hv7nchis8jix0nfc1zjvi9fm6pi2
cannot build derivation '/nix/store/2fvpxgsj9wq03p3y9ia2wix8hh52cqpx-powerdevil-5.18.5.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/qid6assapg1qdi06dvpd4s7hp9qkhn8x-system-path.drv': 1 dependencies couldn't be built
```

**Notify maintainers**

**Metadata**
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

 - system: `"x86_64-linux"`
 - host os: `Linux 5.4.60, NixOS, 20.09pre239987.27ff76db67e (Nightingale)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.3.7`
 - channels(hyper): `"nixos-19.09pre185402.5eac339829b"`
 - channels(root): `"nixos-20.09pre241861.1965a241fc2"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`

Maintainer information:
```yaml
# a list of nixpkgs attributes affected by the problem
attribute:
# a list of nixos modules affected by the problem
module:
```

**Comment by eliasp** [comment]
@ttuegel caused by 24ebfb1e021b60ffa7ccb9970e5d9f0f664faa10

**Comment by eliasp** [comment]
Fixed in c5dcb9682ebb7b586601beb65267a9241df156f8 - not yet in `nixos-unstable-small`.

**Comment by eliasp** [comment]
…and it's in `nixos-unstable-small`.
